explorer/client/src/components/JobRuns/Details.tsx

```diff
@@ -97,7 +97,7 @@ const Details = ({ classes, jobRun }: IProps) => {
         </TableRow>
         <TableRow>
           <KeyCol>Completed At</KeyCol>
-          <Col>{jobRun.completedAt}</Col>
+          <Col>{jobRun.status === 'completed' && jobRun.completedAt}</Col>
```

Comment:
It seems like `jobRun.completedAt` shouldn't be filled in until the status is completed. Is this a field that is automatically filled in, or is this inaccurately sent over from the core?

Reply:
I felt like that was the case. It's getting sent from the core node.